### PR TITLE
Add docs for media player group

### PR DESCRIPTION
**Pull request by definitio**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/38855
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/media_player.group.markdown [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+The group media player platform lets you combine multiple media players into one entity. All child media players of a media player group can still be used as usual, but controlling the state of the grouped media player will forward the command to each child media player.
+
+To enable this platform in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: group
+    name: TVs
```
Comment (klaasnicolaas) [inline source/_integrations/media_player.group.markdown]: Only required options in the first example

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
Please change the file name from `media_player.group.markdown` to `group.markdown`, we don't use platform names in the filename for a long time.

Something in me also says that this is more a part of the media_player integration than a separate thing 🤔  What's your opinion @frenck ?

File: source/_integrations/media_player.group.markdown (hunk 1 above)
Comment (definitio) [reply source/_integrations/media_player.group.markdown]: OK

**Comment by definitio** [comment]
There is `group.markdown` already.

File: source/_integrations/media_player.group.markdown [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: "Media Player Group"
+description: "Instructions for how to setup Media player groups within Home Assistant."
+ha_category:
+  - Media Player
+ha_release: 0.117
```
Comment (frenck) [inline source/_integrations/media_player.group.markdown]:
```suggestion
ha_release: "2021.7"
```

**Review by frenck — APPROVED** [review]
Thanks, @definitio 👍

**Pull request merged**